.github/workflows/main.yml

```diff
@@ -125,16 +125,19 @@ jobs:
             postgres: "13"
             BROWSER: "chrome"
             COVERAGE: 1
+            JOB_COUNT: 2
```

Comment:
This changes the build matrix, which means adjustment is required for the required jobs on branch protection...